src/Components/Components/test/Forms/FieldIdentifierTest.cs

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Components.Forms;
```

Comment:
nit: all of your stuff seems to sort with non-`System.` first but we use `System.` first. I thought we defined an editorconfig for this already..

Reply:
I've never deliberately configured anything about this. I'm just using VS2019 in its out-of-box config. Maybe VS doesn't care about the `.editorconfig`.

I've reordered the namespaces in a bunch of files, hopefully all the new ones.

Reply:
VS is supposed to use `.editorconfig` for this, and yes we do use a non-default setting.